conf/messages.en

```diff
@@ -2506,7 +2507,7 @@ reportAnotherTransferOut.heading = Do you need to report another transfer out fo
 reportAnotherTransferOut.hint = You must tell us about all transfers out for this member during the period of this return.
 reportAnotherTransferOut.error.required = Select yes if you need to report another transfer out for this member
 
-
+checkYourAnswers.transferOutCYA.check.title = Change details of transfers out for this member
```

Comment:
Can this be `transfersOutCYAController.heading.check` please, to be consistent with the lines below?